rpkg_src/hash_probe_from_file.cpp

```diff
@@ -25,6 +22,7 @@ void rpkg_function::hash_probe_from_file(std::string& input_path, std::string& f
 
     if (file::path_exists(input_rpkg_folder_path))
     {
+        // TODO: This should not be hardcoded. Move to some kind of userland module? -RD
```

Comment:
This would've been a left over from development, should be looked into.

Reply:
Hence the to-do comment.

Reply:
Yup!